tests/scripts/test_ingest.py

```diff
@@ -152,6 +152,41 @@ def test_ingest_append(raster_file, tmpworkdir):
     assert all((ds,) in driver.get_datasets() for ds in ('img1', 'img2'))
 
 
+@pytest.mark.parametrize('addflags', [['--skip-existing', ], []])
```

Comment:
I think I would prefer simpler logic here. Something like a boolean `skip_existing` flag.

Reply:
It is a boolean flag. This parametrization just tests both - overwrite with same key when the flag is set and when it is not.

Reply:
I get it now.

Reply:
Done.